### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 105)

### LINKED ISSUE #3439: [devtools-translate] Russian - What's New in DevTools (Chrome 105)
Post: https://developer.chrome.com/blog/new-in-devtools-105/
Translated by: August 19

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

### PR DESCRIPTION
**Pull request by solarrust**
Hi there! 

New translation is ready. 

@webmaxru @kateryna-prokopenko I'll be glad of your review as always ♥️

Closes #3439

### CODE REVIEW

**Comment by jecfish** [comment]
Thanks Alena! Over to @webmaxru and @kateryna-prokopenko for review.

File: site/ru/blog/new-in-devtools-105/index.md [hunk 1]
```diff
@@ -22,178 +21,225 @@ draft: true
...
+Регистрация обработчика протокола URL позволяет установленным PWA обрабатывать ссылки с определённым протоколом (например [`magnet`](https://wikipedia.org/wiki/Magnet_URI_scheme), `web+example`) для более полноценного опыта использования. 
 
 <!-- Navigate to the **Protocol Handlers** section via the **Application** > **Manifest** pane. You can view and test all the available protocols here. -->
+Перейдите в раздел **Обработчики протоколов** (Protocol Handlers) во вкладке **Приложение** (Application) > **Манифест** (Manifest). Вы можете увидеть и протестировать здесь все доступные протоколы.
 
 <!-- For example, install [this demo PWA](https://protocol-handler.glitch.me/). In the **Protocol Handlers** section, type “americano” and click **Test protocol** to open the coffee page in the PWA.  -->
+Например, установите это [тестовое PWA](https://protocol-handler.glitch.me/). В разделе **Обработчики протоколов** (Protocol Handlers) наберите «americano» и нажмите **Тестировать протокол** (Test protocol) чтобы открыть страницу с кофе в PWA.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:109]:
```suggestion
Например, установите это [тестовое PWA](https://protocol-handler.glitch.me/). В разделе **Обработчики протоколов** (Protocol Handlers) введите «americano» и нажмите **Тестировать протокол** (Test protocol), чтобы открыть страницу с кофе в PWA.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 2]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 <!-- The [`<dialog>` element](https://web.dev/building-a-dialog-component/) has recently become stable across browsers. When you open a dialog, it is put into a [top layer](/blog/top-layer-devtools/). Top level content renders on top of all the other content.  -->
+Элемент [`<dialog>`](https://web.dev/building-a-dialog-component/) недавно получил стабильную поддержку во всех браузерах. Когда вы открываете диалог он помещается в [верхний слой](/blog/top-layer-devtools/). Контент верхнего уровня отображается поверх всего остального контента. 
 
 <!-- In this [demo](https://jec.fyi/demo/dialog), click **Open dialog**.  -->
+В этом [демо](https://jec.fyi/demo/dialog) кликните на **Open dialog**.
 
 <!-- To help visualize the top layer elements, DevTools adds a top layer container (`#top-layer`) to the DOM tree. It resides after the closing `</html>` tag.   -->
+Чтобы помочь визуализировать элементы верхнего слоя, в DevTools добавляет контейнер верхнего слоя (`#top-layer`) в DOM-дерево. Он размещается после закрывающего тега `</html>`.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:132]:
```suggestion
Чтобы помочь визуализировать элементы верхнего слоя, DevTools добавляет контейнер верхнего слоя (`#top-layer`) в DOM-дерево. Он размещен после закрывающего тега `</html>`.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 3]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 
 <!-- ## Attach Wasm debugging information at runtime {: #wasm } -->
+## Прикрепление сведений для отладки Wasm во время исполнения {: #wasm }
 
 <!-- You can now attach DWARF debugging information for wasm during runtime. Previously, the **Sources** panel only supported attaching sourcemaps to JavaScript and Wasm files. -->
+Теперь вы можете прикрепить отладочную информацию DWARF для wasm во время исполнения файла. Раньше на вкладке **Источники** (Sources) можно было прикрепить только карту источников для файлов JavaScript и Wasm.
 
 <!-- Open a Wasm file in the **Sources** panel. Right-click in the editor and select **Add DWARF debugging info…**  to attach debugging information on demand.  -->
+Откройте файл Wasm во вкладке **Источники** (Sources). Кликните правой клавишей мыши в области редактора и выберите **Добавить сведения для отладки DWARF…** (Add DWARF debugging info…) чтобы прикрепить отладочную информацию.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:154]:
```suggestion
Откройте файл Wasm во вкладке **Источники** (Sources). Кликните правой клавишей мыши в области редактора и выберите **Добавить сведения для отладки DWARF…** (Add DWARF debugging info…) чтобы прикрепить отладочную информацию по запросу.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 4]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
-Chromium issue: [1341255](https://crbug.com/1341255)
+Задача в трекере Chromium: [1341255](https://crbug.com/1341255)
 
 
 <!-- ## Support live edit during debugging {: #live-edit } -->
+## Поддержка редактирования во время отладки {: #live-edit }
 
 <!-- You can now edit the top-most function on the stack without restarting the debugger. -->
+Теперь вы можете редактировать самую верхнюю функцию в стеке без необходимости перезапускать отладчик.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:167]:
```suggestion
Теперь вы можете редактировать самую верхнюю функцию в стеке без перезапуска отладчика.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 5]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 <!-- In Chrome 104, DevTools brings back the [restart frame](/blog/new-in-devtools-104/) feature. However, you weren't able to edit the function you are currently paused in. It is common for developers to break in a function and then edit that function while paused.  -->
+В Chrome 104 в DevTools вернулась возможность [перезапуска фрейма](/blog/new-in-devtools-104/). Тем не менее вы не могли редактировать функцию, на которой вы остановились прямо сейчас. Обычно разработчики прерывают функцию, а затем редактируют ее во время паузы.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:170]:
```suggestion
В Chrome 104 в DevTools вернулась возможность [перезапуска фрейма](/blog/new-in-devtools-104/). Тем не менее вы не могли редактировать функцию, на которой вы остановились прямо сейчас. Разработчики часто прерывают функцию, а затем редактируют ее во время паузы.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 6]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 <!-- With this update, the debugger automatically restarts the function with the following restrictions: -->
+С этим обновлением отладчик автоматически перезапускает функцию при следующих ограничениями:
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:173]:
```suggestion
С этим обновлением отладчик автоматически перезапускает функцию со следующими ограничениями:
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 7]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 <!-- - Only the top-most function can be edited while paused -->
 <!-- - No recursive call on the same function further down the stack -->
+- Только функция верхнего уровня может быть отредактирована во время паузы.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:177]:
```suggestion
- Только функция самого верхнего уровня может быть отредактирована во время паузы.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 8]
```diff
@@ -22,178 +21,225 @@ draft: true
...
+- Ниже по стеку нет рекурсивных вызовов одной и той же функции.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-105/index.md:178]:
```suggestion
- Ниже по стеку нет рекурсивных вызовов этой же функции.
```

**Review by kateryna-prokopenko — COMMENTED** [review]
Thanks Alena! Left some comments

**Comment by solarrust** [comment]
@kateryna-prokopenko really appreciate for your review! 🙌

File: site/ru/blog/new-in-devtools-105/index.md [hunk 9]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/NZZJ1FzXxqj2U2NR0U53.png", alt="LCP in the Performance insights panel", width="800", height="751" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/NZZJ1FzXxqj2U2NR0U53.png", alt="LCP во вкладке Performance Insights", width="800", height="751" %}
 
-Chromium issue: [1326481](https://crbug.com/1326481)
+Задача в трекере Chromium: [1326481](https://crbug.com/1326481)
 
 
 <!-- ## Identify flashes of text (FOIT, FOUT) as potential root causes for layout shifts {: #foit-fout } -->
+## Определение вспышки текста (FOIT, FOUT) как потенциальных первопричин сдвигов раскладки {: #foit-fout }
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:80]:
```suggestion
## Определение миганий текста (FOIT, FOUT) как потенциальных первопричин сдвигов раскладки {: #foit-fout }
```

File: site/_includes/partials/devtools/ru/whats-new.md [hunk 10]
```diff
@@ -4,18 +4,19 @@
...
-* [Support live edit during debugging](/ru/blog/new-in-devtools-105/#live-edit)
-* [View and edit @scope at rules in the Styles pane](/ru/blog/new-in-devtools-105/#scope)
-* [Sourcemap improvements](/ru/blog/new-in-devtools-105/#sourcemaps)
-* [Miscellaneous highlights](/ru/blog/new-in-devtools-105/#misc) -->
+### Chrome 105 {: #chrome105 }
+
+* [Пошаговое воспроизведение в Recorder](/ru/blog/new-in-devtools-105/#recorder)
+* [Поддержка события mouse over во вкладке Recorder](/ru/blog/new-in-devtools-105/#recorder-hover)
+* [Самое крупное существенное отображение (LCP) во вкладке Performance Insights](/ru/blog/new-in-devtools-105/#lcp)
+* [Определение вспышки текста (FOIT, FOUT) как потенциальных первопричин сдвигов раскладки](/ru/blog/new-in-devtools-105/#foit-fout)
```
Comment (webmaxru) [inline site/_includes/partials/devtools/ru/whats-new.md:12]:
```suggestion
* [Определение миганий текста (FOIT, FOUT) как потенциальных первопричин сдвигов раскладки](/ru/blog/new-in-devtools-105/#foit-fout)
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 11]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 <!-- The **Performance insights** panel now detects [flash of invisible text (FOIT) and flash of unstyled text (FOUT)](https://web.dev/preload-optional-fonts/#font-rendering) as potential root causes for layout shifts. -->
+Во вкладке **Performance Insights** теперь распознаются [вспышки невидимого текста (FOIT) и вспышки нестилизованного текста (FOUT)](https://web.dev/preload-optional-fonts/#font-rendering) как потенциальные причины сдвига раскладки.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:83]:
```suggestion
Во вкладке **Performance Insights** теперь распознаются [мигания невидимого текста (FOIT) и мигания нестилизованного текста (FOUT)](https://web.dev/preload-optional-fonts/#font-rendering) как потенциальные причины сдвига раскладки.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 12]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 <!-- To view the potential root causes of a layout shift, click on a screenshot in the **Layout shifts** track. -->
+Чтобы увидеть потенциальные причины сдвига раскладки кликните на скриншот в **Layout shifts**.
 
 <!-- See [Optimize WebFont loading and rendering](https://web.dev/optimize-webfont-loading/) to learn the technique to prevent layout shifts.  -->
+Читайте [Optimize WebFont loading and rendering](https://web.dev/optimize-webfont-loading/) чтобы узнать о техниках предотвращения сдвигв раскладки.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:89]:
```suggestion
Изучите статью [Optimize WebFont loading and rendering](https://web.dev/optimize-webfont-loading/), чтобы узнать о техниках предотвращения сдвига раскладки.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 13]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
-Chromium issues: [1300613](https://crbug.com/1300613)
+Задачи в трекере Chromium: [1300613](https://crbug.com/1300613)
 
 
 <!-- ## Top layer badge in the Elements panel {: #top-layer } -->
+## Значок верхнего слоя во вкладке Элементы {: #top-layer }
 
 <!-- Use the [top layer badge](/blog/top-layer-devtools/#top-layer-support-design-in-devtools) to understand the concept of the top layer and visualize how the top layer content changes.  -->
+Используйте [значок верхнего слоя](/blog/top-layer-devtools/#top-layer-support-design-in-devtools), чтобы понять концепцию верхнего слоя и визуализировать как меняется его контент.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:123]:
```suggestion
Используйте [значок верхнего слоя](/blog/top-layer-devtools/#top-layer-support-design-in-devtools), чтобы понять концепцию верхнего слоя и визуализировать, как меняется его контент.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 14]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 
 <!-- ## Attach Wasm debugging information at runtime {: #wasm } -->
+## Прикрепление сведений для отладки Wasm во время исполнения {: #wasm }
 
 <!-- You can now attach DWARF debugging information for wasm during runtime. Previously, the **Sources** panel only supported attaching sourcemaps to JavaScript and Wasm files. -->
+Теперь вы можете прикрепить отладочную информацию DWARF для wasm во время исполнения файла. Раньше на вкладке **Источники** (Sources) можно было прикрепить только карту источников для файлов JavaScript и Wasm.
 
 <!-- Open a Wasm file in the **Sources** panel. Right-click in the editor and select **Add DWARF debugging info…**  to attach debugging information on demand.  -->
+Откройте файл Wasm во вкладке **Источники** (Sources). Кликните правой клавишей мыши в области редактора и выберите **Добавить сведения для отладки DWARF…** (Add DWARF debugging info…) чтобы прикрепить отладочную информацию по запросу.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:154]:
```suggestion
Откройте файл Wasm во вкладке **Источники** (Sources). Кликните правой клавишей мыши в области редактора и выберите **Добавить сведения для отладки DWARF…** (Add DWARF debugging info…), чтобы прикрепить отладочную информацию по запросу.
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 15]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 <!-- With this update, the debugger automatically restarts the function with the following restrictions: -->
+С этим обновлением отладчик автоматически перезапускает функцию со следующими ограничениями:
 
 <!-- - Only the top-most function can be edited while paused -->
 <!-- - No recursive call on the same function further down the stack -->
+- Только функция самого верхнего уровня может быть отредактирована во время паузы.
+- Ниже по стеку нет рекурсивных вызовов этой же функции.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0PG2PnQUh5bnpIulyj7m.png", alt="live edit during debugging", width="800", height="560" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0PG2PnQUh5bnpIulyj7m.png", alt="редактирование во время отладки", width="800", height="560" %}
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:180]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/0PG2PnQUh5bnpIulyj7m.png", alt="Редактирование во время отладки", width="800", height="560" %}
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 16]
```diff
@@ -22,178 +21,225 @@ draft: true
...
+Откройте эту [демо-страницу](https://codepen.io/miriamsuzanne/details/ZErXZVY) и проинспектируйте гиперссылку внутри элемента `<div class="dark-theme">`. В панели **Стили** (Styles) просмотрите директиву `@scope`. Кликните на объявление правила, чтобы отредактировать его.
 
 {% Aside %}
 <!-- The CSS `@scope` is currently under development. To test this feature, enable the **Experimental Web Platform features** flag via `chrome://flags/#enable-experimental-web-platform-features`. -->
+Поддержка `@scope` в настоящее время находится в стадии разработки. Чтобы протестировать эту функцию, включите флаг **Experimental Web Platform features** на странице `chrome://flags/#enable-experimental-web-platform-features`.
 {% endAside %}
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LnkBUWoEl11HGiAD4ag7.png", alt="@scope at rules in the Styles pane", width="800", height="464" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LnkBUWoEl11HGiAD4ag7.png", alt="директива @scope на вкладке Стили", width="800", height="464" %}
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:204]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/LnkBUWoEl11HGiAD4ag7.png", alt="Директива @scope на вкладке Стили", width="800", height="464" %}
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 17]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
 <!-- Here are a few fixes on sourcemaps to improve the overall debugging experience: -->
+Вот несколько исправлений в картах источников для улучшения общего опыта отладки:
 
 <!-- - DevTools now properly resolves sourcemap identifiers with punctuation. Some modern minifiers (for example, [esbuild](https://esbuild.github.io/)) produce sourcemaps that merge identifiers with subsequent punctuation (comma, parentheses, semicolon).  -->
+- DevTools теперь правильно распознает идентификаторы карт источников с пунктуацией. Некоторые современные минификаторы (например, [esbuild](https://esbuild.github.io/)) создают карты источников, объединяющие идентификаторы с помощью знаков препинания (запятая, круглые скобки, точка с запятой).
 <!-- - DevTools now resolves sourcemap names for constructors with a `super` call. -->
-  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/6djFfkrtPzXuNYq5m8Vk.png", alt="ALT_TEXT_HERE", width="800", height="441" %}
+- DevTools теперь распознает имена карт источников для конструкторов с вызовом `super`. 
+  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/6djFfkrtPzXuNYq5m8Vk.png", alt="правильное распознание вызова super", width="800", height="441" %}
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:221]:
```suggestion
  {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/6djFfkrtPzXuNYq5m8Vk.png", alt="Правильное распознание вызова super", width="800", height="441" %}
```

File: site/ru/blog/new-in-devtools-105/index.md [hunk 18]
```diff
@@ -22,178 +21,225 @@ draft: true
...
 
-Chromium issue: [1257499](https://crbug.com/1257499)
+Задача в трекере Chromium: [1257499](https://crbug.com/1257499)
 
 
 <!-- ## Largest Contentful Paint (LCP) in the Performance insights panel {: #lcp } -->
+## Самое крупное существенное отображение (LCP) во вкладке Performance Insights {: #lcp }
 
 <!-- LCP is an important, user-centric metric for measuring [perceived load speed](https://web.dev/user-centric-performance-metrics/#types-of-metrics). You can now find out the critical paths and root causes of a [Largest Contentful Paint (LCP)](https://web.dev/lcp/). -->
+LCP это важная, ориентированная на пользователя метрика, замеряющая [ощущаемую скорость нагрузки](https://web.dev/user-centric-performance-metrics/#types-of-metrics). Теперь вы можете отследить критические моменты и первопричины задержки отрисовки [крупного существенного отображения (LCP)](https://web.dev/lcp/).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-105/index.md:66]:
```suggestion
LCP – это важная, ориентированная на пользователя метрика, замеряющая [ощущаемую скорость нагрузки](https://web.dev/user-centric-performance-metrics/#types-of-metrics). Теперь вы можете отследить критические моменты и первопричины задержки отрисовки [крупного существенного отображения (LCP)](https://web.dev/lcp/).
```

**Comment by jecfish** [comment]
Thanks @webmaxru . Over to @solarrust for final review

**Comment by solarrust** [comment]
@webmaxru thank you for your review! All fixed. 

@jecfish seems like done.

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Merge it, thanks all!

**Pull request merged**